Index: src/main/java/com/ramiaran/heritagesitesystem/HeritageController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.ramiaran.heritagesitesystem;\n\nimport javafx.fxml.FXML;\nimport javafx.scene.control.Label;\n\npublic class HeritageController {\n    @FXML\n    private Label welcomeText;\n\n    @FXML\n    protected void onHelloButtonClick() {\n        welcomeText.setText(\"Welcome to JavaFX Application!\");\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/ramiaran/heritagesitesystem/HeritageController.java b/src/main/java/com/ramiaran/heritagesitesystem/HeritageController.java
--- a/src/main/java/com/ramiaran/heritagesitesystem/HeritageController.java	(revision 6bfdf942119a883cff393bbb8e5ee10efd8c1fd1)
+++ b/src/main/java/com/ramiaran/heritagesitesystem/HeritageController.java	(date 1766674865727)
@@ -11,4 +11,4 @@
     protected void onHelloButtonClick() {
         welcomeText.setText("Welcome to JavaFX Application!");
     }
-}
\ No newline at end of file
+}
